data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml

```diff
@@ -4,30 +4,31 @@
 <teiHeader xml:lang="eng">
 <fileDesc>
 <titleStmt>
-<title xml:lang="eng">The Cock</title>
+<title>The Cock</title>
  <author>Lucian of Samosata</author>
 <editor role="translator">Emily James Smith</editor>
  <funder>National Endowment for the Humanities</funder>
  <principal>Gregory Crane</principal>
  <respStmt>
-  <persName>Gregory Crane</persName>
-<resp>Digital conversion and editing</resp>
+  <resp>Prepared under the supervision of</resp>
```

Comment:
I see this added here but not in the rest of the batch?

Reply:
Yes, I realized I was getting too bogged down in the Emily James Smith files and decided to do a separate pass on those to fix both cts and bigger edits (per your other issue!)

Reply:
ok!